README.md

```diff
@@ -17,7 +17,7 @@ This project is a command-line to-do list manager, where users can add, view, an
 ### Set up your repository
 1. Clone the repository to your local machine
 2. Navigate to the project directory (using `cd` in the terminal)
-3. Run the project: `python todo.py`
+3. Run the project: `python3 todo.py`
```

Comment:
Okay! python3 should work. However, we need to add some more comments to explain this!